includes/class-main.php

```diff
@@ -0,0 +1,172 @@
+<?php
```

Comment:
Move the changes to this file over to the class-coding-challenge.php, and delete this file